api/docs/release.dox

```diff
@@ -141,6 +141,9 @@ Further non-compatibility-affecting changes include:
    -replay_file, -cpu_schedule_file).
```

Comment:
> …cords matching operands.

Noticed that this is still in the description.

Reply:
Thanks for catching it.